addons/website/models/res_config_settings.py

```diff
@@ -121,6 +121,8 @@ def _default_website(self):
     module_website_livechat = fields.Boolean()
     module_marketing_automation = fields.Boolean()
 
+    module_website_sale_autocomplete = fields.Boolean('Address Autocomplete')
+
```

Comment:
Actually shouldn't this change be in `website_sale` instead?
Since it is only used for `website_sale`